iohk-nix.json

```diff
@@ -1,7 +1,7 @@
 {
   "url": "https://github.com/input-output-hk/iohk-nix",
-  "rev": "a2a976a26119662756327e278b6ca9cd20e2f6f0",
```

Comment:
This needs to be a commit on the master branch.

Reply:
it will be once https://github.com/input-output-hk/iohk-nix/pull/84 is merged